Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -68,7 +68,8 @@ data DSearchReq = DSearchReq
     dropLocation :: LatLong,
     routeDistance :: Maybe Meters,
     routeDuration :: Maybe Seconds,
-    device :: Maybe Text
+    device :: Maybe Text,
+    parentSearchId :: Maybe Text
```

Comment:
I think `parentTransactionId` would be more clear

Reply:
done

Reply:
@DonDokidon
`searchId` on bap side is the same as `transactionId` on bpp side
`parentSearchId` on bap side is the same as `parentTransactionId` on bpp side
So it would be consistently to call it `parentTransactionId`, I guess

Reply:
@DonDokidon
`searchId` on bap side is the same as `transactionId` on bpp side
`parentSearchId` on bap side is the same as `parentTransactionId` on bpp side
So it would be consistently to call it `parentTransactionId`, I guess